enclave-runtime/src/lib.rs

```diff
@@ -696,48 +700,43 @@ where
 	Ok(())
 }
 
-fn sync_blocks_on_light_client<PB, V, OCallApi, StateHandler>(
+fn sync_blocks_on_light_client<PB, V, OCallApi, StfExecutor>(
 	blocks_to_sync: Vec<SignedBlockG<PB>>,
 	validator: &mut V,
 	on_chain_ocall_api: &OCallApi,
-	state_handler: &StateHandler,
+	stf_executor: &StfExecutor,
```

Comment:
In the `enclave-runtime` the `StfExecutor` now replaces the `StateHandler` and thus hiding the state handler details behind the executor.